cts/scheduler/summary/promoted-demote.summary

```diff
@@ -7,13 +7,13 @@ Current cluster status:
     * cyrus_master	(ocf:heartbeat:cyrus-imap):	 Stopped
     * cyrus_syslogd	(ocf:heartbeat:syslogd):	 Stopped
     * cyrus_filesys	(ocf:heartbeat:Filesystem):	 Stopped
-    * cyrus_volgroup	(ocf:heartbeat:VolGroup):	 Stopped
+    * cyrus_volgroup	(ocf:heartbeat:VolGroup):	 Stopped (not installed) 
```

Comment:
This suggests we had a bug in the display code previously :-/

Hopefully requiring on_node avoids whatever it was ...

Reply:
In native.c:
```
    // Failed probe operation
    if (native_displayable_role(rsc) == pcmk_role_stopped) {
        xmlNode *probe_op = pe__failed_probe_for_rsc(rsc, node ? node->private->name : NULL);
        if (probe_op != NULL) {
            int rc;

            pcmk__scan_min_int(crm_element_value(probe_op, PCMK__XA_RC_CODE),
                               &rc, 0);
            pcmk__g_strcat(outstr, " (", services_ocf_exitcode_str(rc), ") ",
                           NULL);
        }
    }
```

and similar in clone.c

`pe__failed_probe_for_rsc()` skips an `lrm_rsc_op` if it's missing the `on_node` attribute.

Reply:
You resolved this and approved the PR so I assume it's fine for now, but feel free to create a task if you feel that this needs addressing.

Reply:
It's fine for `pe__failed_probe_for_rsc()` to require on_node